pkg/store/zookeeper.go

```diff
@@ -115,7 +115,7 @@ func (s *Zookeeper) Watch(key string, _ time.Duration, callback WatchCallback) e
 			entry, index, err := s.Get(key)
 			kvi := []KVEntry{&kviTuple{key, []byte(entry), index}}
 			if err == nil {
-				callback(kvi)
```

Comment:
Minor comment. It would be cool to remove the array here as well (just like you did for consul and etcd). 
otherwise LGTM.